src/project-tests/pomodoro-clock-tests.js

```diff
@@ -823,19 +823,18 @@ export default function createPomodoroClockTests() {
       be rewound to the beginning when the element with the id of "reset" is
       clicked.`,
       function() {
-        var audioElem = document.getElementById('beep');
-
+        const audioElem = document.getElementById('beep');
         audioElem.play();
         resetTimer();
 
         assert.isTrue(
-          audioElem.paused,
+          document.getElementById('beep').paused,
           'Audio element was not stopped when reset was clicked.'
         );
 
         assert.equal(
           0,
-          audioElem.currentTime,
+          document.getElementById('beep').currentTime,
```

Comment:
@tchaffee The pomodoro test was not passing without this change, and this PR is needed to pass subsequent PRs, so consider allowing this non-sequitur change. In my understanding, doing .getElementById each test helps bust the component's cache...

Reply:
I'd rather see this as a separate PR. The problem is that in the future when someone new to the project looks to reverse this PR then they see changes to Pomodoro tests and they don't know whether or not to reverse these seemingly related changes too. If you are tight on time and need help I can create a separate PR for these Pomodoro improvements.

Reply:
OK, #312 passes local Selenium tests.